### ISSUE TITLE
Class 'FlutterLocationService' is not abstract and does not implement abstract member public abstract fun onRequestPermissionsResult

### ISSUE THREAD

**Issue by Yczar** [Lyokone/flutterlocation#720]
**Describe the bug**
After upgrading to flutter 3.0.0 I got this error in my console

**Expected behavior**
It is expected to run smoothly like it has always done.

**Tested on:**
 - Android, API Level XX [e.g. 30], simulator and real device

``` shell
 /Users/czar/development/flutter/.pub-cache/hosted/pub.dartlang.org/location-4.3.0/android/src/main/java/com/lyokone/location/FlutterLocationService.kt: (124, 1): Class 'FlutterLocationService' is not abstract and does not implement abstract member public abstract fun onRequestPermissionsResult(p0: Int, p1: Array<(out) String!>, p2: IntArray): Boolean defined in io.flutter.plugin.common.PluginRegistry.RequestPermissionsResultListener
e: /Users/czar/development/flutter/.pub-cache/hosted/pub.dartlang.org/location-4.3.0/android/src/main/java/com/lyokone/location/FlutterLocationService.kt: (258, 5): 'onRequestPermissionsResult' overrides nothing
```

**Comment by Yczar** [comment]
Here is a PR for the maintainers to test. I fixed the above Issue here https://github.com/Lyokone/flutterlocation/pull/721

**Comment by Yczar** [comment]
And for the main time you can use my updated git package this way

```
location:
    git:
      url: https://github.com/Yczar/flutterlocation.git
      path: packages/location
```

**Comment by bayuly94** [comment]
> And for the main time you can use my updated git package this way
> 
> ```
> location:
>     git:
>       url: https://github.com/Yczar/flutterlocation.git
>       path: packages/location
> ```

thanks

**Comment by saravanans55** [comment]
@bayuly94 @Yczar 
Hi ,
Can u pls tell me how to use this to fix this issue? where i should replace it?

``location:
    git:
      url: https://github.com/Yczar/flutterlocation.git
      path: packages/location``

**Comment by Yczar** [comment]
> @bayuly94 @Yczar Hi , Can u pls tell me how to use this to fix this issue? where i should replace it?
> 
> `location: git: url: https://github.com/Yczar/flutterlocation.git path: packages/location`

Replace with you location dependency in your pubspec

**Comment by Lyokone** [comment]
It's published in 4.4.0.
Thanks for your help on this.
I don't have a lot of time to maintain the package right now, but it should improve soon 👀
